src/read_dicom_file.py

```diff
@@ -0,0 +1,41 @@
+"""
+provides a read_dicom_file() function that takes a file path to a dicom file, checks if the
+file is valid, and returns the FileDataset object. If the file is not valid it will return None.
+"""
+
+import logging
+import pydicom
+from pydicom.errors import InvalidDicomError
+
+logger = logging.getLogger(__name__)  # Start logger
+
+
+def read_dicom_file(file_path):
```

Comment:
recommend using typing for inputs and outputs.
Include inputs and outputs as part of  docstring.  There are a few idioms for that for docstring.